Paths fixed in the patch- dilfridge@gentoo.org

From 770933ee2c194160a9ab3320468d3b672973d3d8 Mon Sep 17 00:00:00 2001
From: Peter Mottram <peter@sysnix.com>
Date: Thu, 12 May 2016 16:50:35 +0200
Subject: [PATCH] fix breakage introduced in MIME::Lite v3.029

See commit:

https://github.com/rjbs/MIME-Lite/commit/4608623af9126f53f38b723c4e9f38bbcd07d866

and original ticket:

https://rt.cpan.org/Public/Bug/Display.html?id=35979
---
 lib/MIME/Lite/HTML.pm | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lib/MIME/Lite/HTML.pm b/lib/MIME/Lite/HTML.pm
index bc06640..6839bf4 100755
--- a/HTML.pm
+++ b/HTML.pm
@@ -463,7 +463,6 @@ sub build_mime_object {
 			  'Data'     => $html);
     $part->attr("content-type"=> "text/html; charset=".$self->{_htmlcharset});
     # Remove some header for Eudora client in HTML and related part
-    $part->replace("MIME-Version" => "");
     $part->replace('X-Mailer' =>"");
     $part->replace('Content-Disposition' =>"");
     # only html, no images & no txt
@@ -480,7 +479,6 @@ sub build_mime_object {
     $txt_part->attr("content-type" => 
 		    "text/plain; charset=".$self->{_textcharset});
     # Remove some header for Eudora client
-    $txt_part->replace("MIME-Version" => "");
     $txt_part->replace("X-Mailer" => "");
     $txt_part->replace("Content-Disposition" => "");
     # only text, no html
@@ -516,7 +514,6 @@ sub build_mime_object {
     # Create related part
     my $rel = new MIME::Lite ('Type'=>'multipart/related');
     $rel->replace("Content-transfer-encoding" => "");
-    $rel->replace("MIME-Version" => "");
     $rel->replace("X-Mailer" => "");
     # Attach text part to alternative part
     $mail->attach($txt_part);
@@ -656,7 +653,6 @@ sub create_image_part {
 
   # Remove header for Eudora client
   $mail->replace("X-Mailer" => "");
-  $mail->replace("MIME-Version" => "");
   $mail->replace("Content-Disposition" => "");
   return $mail;
 }
